apps/web/src/app/auth/login/login-with-device.component.html

```diff
@@ -5,42 +5,71 @@
 >
   <div>
     <img class="logo logo-themed" alt="Bitwarden" />
-    <p class="tw-mx-4 tw-mb-4 tw-mt-3 tw-text-center tw-text-xl">
-      {{ "loginOrCreateNewAccount" | i18n }}
-    </p>
 
-    <div
-      class="tw-mt-3 tw-rounded-md tw-border tw-border-solid tw-border-secondary-300 tw-bg-background tw-p-6"
-    >
-      <h2 class="tw-mb-6 tw-text-xl tw-font-semibold">{{ "logInInitiated" | i18n }}</h2>
+    <ng-container *ngIf="state == State.StandardAuthRequest">
+      <p class="tw-mx-4 tw-mb-4 tw-mt-3 tw-text-center tw-text-xl">
+        {{ "loginOrCreateNewAccount" | i18n }}
+      </p>
 
-      <div class="tw-text-light">
-        <p class="tw-mb-6">{{ "notificationSentDevice" | i18n }}</p>
+      <div
+        class="tw-mt-3 tw-rounded-md tw-border tw-border-solid tw-border-secondary-300 tw-bg-background tw-p-6"
+      >
+        <h2 class="tw-mb-6 tw-text-xl tw-font-semibold">{{ "logInInitiated" | i18n }}</h2>
 
-        <p class="tw-mb-6">
-          {{ "fingerprintMatchInfo" | i18n }}
-        </p>
-      </div>
+        <div class="tw-text-light">
+          <p class="tw-mb-6">{{ "notificationSentDevice" | i18n }}</p>
 
-      <div class="tw-mb-6">
-        <h4 class="tw-font-semibold">{{ "fingerprintPhraseHeader" | i18n }}</h4>
-        <p>
-          <code>{{ fingerprintPhrase }}</code>
-        </p>
-      </div>
+          <p class="tw-mb-6">
+            {{ "fingerprintMatchInfo" | i18n }}
+          </p>
+        </div>
+
+        <div class="tw-mb-6">
+          <h4 class="tw-font-semibold">{{ "fingerprintPhraseHeader" | i18n }}</h4>
+          <p>
+            <code>{{ fingerprintPhrase }}</code>
+          </p>
+        </div>
+
+        <div class="tw-my-10" *ngIf="showResendNotification">
+          <a [routerLink]="[]" disabled="true" (click)="startPasswordlessLogin()">{{
```

Comment:
question: why is this always disabled?

suggestion(non-blocking): can be simplified
```suggestion
          <a [routerLink]="[]" disabled (click)="startPasswordlessLogin()">{{
```

Reply:
It's because it was created with the `routerlink` to give it the display of a link, but the creator didn't want the user to be taken to the root of the app on click.